docker/pythonpath_dev/superset_config.py

```diff
@@ -40,6 +40,7 @@
 EXAMPLES_HOST = os.getenv("EXAMPLES_HOST")
 EXAMPLES_PORT = os.getenv("EXAMPLES_PORT")
 EXAMPLES_DB = os.getenv("EXAMPLES_DB")
+SHOW_STACKTRACE = True
```

Comment:
Was this intended to be part of the PR, or just for local/temporary dev purposes?

Reply:
Let me remove that line, now thinking it may make sense to deprecate this flag as it's not evenly implemented.